misaligned.js

```diff
@@ -1,16 +1,25 @@
-const {expect} = require('chai')
 
-function print_color_map() {
+function getColorMap() {
     const majorColors = ["White", "Red", "Black", "Yellow", "Violet"];
     const minorColors = ["Blue", "Orange", "Green", "Brown", "Slate"];
+    let colorMap=[];
     for (let i = 0; i < majorColors.length; i++) {
         for (let j = 0; j < minorColors.length; j++) {
-            console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);
+            //console.log(`${i * 5 + j} | ${majorColors[i]} | ${minorColors[j]}`);

```

Comment:
Don't leave commented code in commits. Remove the code.
If you want to see the older code, it will be there in git

Reply:
Ok sir